src/frontend/templates/pages/home.html

```diff
@@ -1,448 +1,14 @@
 {% extends 'base.html' %}
 
 {% load i18n %}
+{% load custom_formatting %}
 {% load custom_translation %}
 
 {% block extra_html_head_stuff %}
-<link type="text/css" href="/static/css/elo-graph.css" rel="stylesheet" />
 <link type="text/css" href="/static/css/tooltips.css" rel="stylesheet" />
 
-<script type="text/javascript" src="/static/js/d3.v6.min.js"></script>
-
 {% if run %}
-{% url 'networks-for-elo-list' as networks_for_elo_list_url %}
```

Comment:
nice

Reply:
Factored this out so that we could reuse it both on home page and on runs view.